### DISCUSSION TITLE
Common Discussions

### DISCUSSION THREAD

**Discussion by sudarshan-doiphode** [provectus/kafka-ui#4470] [General]
I am a Springboot developer, How I can contribute on this? Let me know if anything is there for me.

**Comment by Haarolean** [comment]
hi, this repo is not maintained (#4255). Happy to see your contributions here: https://github.com/kafbat/kafka-ui
You can start with the [contributing guide](https://github.com/kafbat/kafka-ui/blob/main/.github/CONTRIBUTING.md)

**Comment by BILL-CHEPTOYEK** [comment]
Explore and contribute to open-source libraries and tools that complement Spring Boot including libraries for security, testing and monitoring
